server/planning/utils.py

```diff
@@ -17,6 +17,10 @@ class FormattedContact(TypedDict):
     website: str
 
 
+ALL_DAY_SECONDS = 86460
+MULTI_DAY_SECONDS = 86400
```

Comment:
it still looks strange to have multi day being lower than all day :sweat_smile: , it can use some doc string

Reply:
you are right! Thank you for pointing it out! 😊